README.md

```diff
@@ -83,26 +83,48 @@ The same process is followed for the asset identifier.
 The identifier is decomposed into its namespace, reference, and optional ID.
 Then, the resolver for the specified asset namespace is used to resolve the asset reference and ID.
 
+### JSON-LD Context
+
+The snippet below shows the context that the process of AssetDID resolution returns.
+
+```json
+{
+  "@protected": true,
+  "id": "@id",
+  "asset-did-spec": "https://github.com/KILTprotocol/spec-asset-did/blob/main/README.md",
+  "chain": {
+    "@id": "asset-did-spec:chain",
```

Comment:
you could add ` "@type":"@id",` for completeness, tells the processor that we expect the value of `chain` to be an IRI. Just covers the case where you'd have `"chain": "eip155:0"` in your document so yeah, just for completeness.